mask/eolearn/mask/cloud_mask.py

```diff
@@ -438,9 +446,11 @@ def _update_batches(
 
         var_data = self._map_sequence(data, local_var_func)
 
-        if local_avg is None:
+        if local_avg is None or local_var is None:
             local_avg = avg_data / avg_data_mask
+            local_avg = cast(np.ndarray, local_avg)
```

Comment:
Is this needed to drop the optional part?

Reply:
yeah, because we are reusing names. And `local_avg` is `Optional[np.ndarray]` (because the condition is `or`) and `mypy` doesn't want to specialize that. We can avoid that by using different names. I didnt feel like thinking about that too hard, because the whole procedure is trash.